.github/rsi-schema.json

```diff
@@ -1,185 +1,209 @@
 {
-   "$schema":"http://json-schema.org/draft-07/schema",
-   "default":{
-      
-   },
-   "description":"JSON Schema for SS14 RSI validation.",
-   "examples":[
-      {
-         "version":1,
-         "license":"CC-BY-SA-3.0",
-         "copyright":"Taken from CODEBASE at COMMIT LINK",
-         "size":{
-            "x":32,
-            "y":32
-         },
-         "states":[
-            {
-               "name":"basic"
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "default": {
+
+    },
+    "description": "JSON Schema for SS14 RSI validation.",
+    "examples": [
+        {
+            "version": 1,
+            "license": "CC-BY-SA-3.0",
+            "copyright": "Taken from CODEBASE at COMMIT LINK",
```

Comment:
would be better to note that those fields are not to be used together with the per-file option. I think it would be better to require one or the other, and NOT allow mixing this. "Default for the file" will get messy easily, and will lead to people forgetting to change things. Though yeah it adds extra work if someone wants to add a new file into existing RSI.

It can too easily lead to confusion in terms of "which applies?". Best to deprecate this.

In general, I am unsure if it's better to go for having a separate array for licensing/copyright that follows RGA format, or if it should be part of the image states array.

Also missing RGA's `links` property. Please use that instead of "taken from COMMIT LINK".

Reply:
Still need to move this over to use `links` :)